test/quantization/test_quantize_fx.py

```diff
@@ -1527,11 +1527,13 @@ def forward(self, x):
 
         m = convert_fx(m)
         keys = m.state_dict().keys()
-
-        scale_count = 0
-        zero_point_count = 0
+        quant_scale_count = quant_zero_point = scale_count = zero_point_count = 0
         for k in keys:
-            if 'scale' in k:
+            if 'quant_scale' in k:
```

Comment:
nit: `quant_scale` is a bit vague, shall we rename it to `input_scale`? same for `zero_point`